transcripts/how-to-apply-for-a-federal-job-with-the-cfpb/index.html

```diff
@@ -0,0 +1,102 @@
+{% extends "layout-full.html" %}
+
+{% block title -%}
+    How to Apply For a Federal Job with the CFPB
+{%- endblock %}
+
+{% block desc -%}
+    Transcript for the video "How to Apply For a Federal Job with the CFPB."
+{%- endblock %}
+
+{% block content_modifiers -%}
+    {{ super() }}
+    transcripts
```

Comment:
This could be removed till we actually need a class for this page. 
Or alternately could we name this `careers-transcripts` or `careers-app-process-transcripts` so the class name is less generic and has some indication of where it is used.